summits/2019-03-07_2019-03-08.md

```diff
@@ -0,0 +1,40 @@
+# Node.js Diagnostics Summit - Spring 2019
+
+- **When:** 2019-03-07 to 2019-03-08 
+- **Where:** Munich, Germany
+- **Issue:** [#203](https://github.com/nodejs/diagnostics/issues/203)
+- **Agenda:** [Google Doc](https://docs.google.com/document/d/1Ub4kqznhEXHzKtS4ArK5KGBzwRLa29Kamr4lo2BvzcE/preview)
+
+## Minutes
+
+Besides the action times linked in this section, there aren't a full set of minutes. For the best overview see the Recordings section below.
+
+- [Action items](https://docs.google.com/document/d/1ftJCXFM_Yi45O2nr5O7YwdOMO7iUbWSx-UStvOYMu28/edit)
```

Comment:
I think it would be good to pull the content from that google doc into the doc being added by this PR instead of being a link.

Reply:
Updated